RiotSwiftUI/Modules/UserSessions/UserSessionsOverview/UserSessionsOverviewModels.swift

```diff
@@ -19,7 +19,7 @@ import Foundation
 // MARK: - Coordinator
 
 enum UserSessionsOverviewCoordinatorResult {
-    case openSessionDetails(session: UserSessionInfo)
+    case openSessionOverview(session: UserSessionInfo, isCurrentSession: Bool)
```

Comment:
I wonder if would be cleaner to move `isCurrentSession` to the `UserSessionInfo`. Otherwise perhaps we should extract the overviewService into the flowsCoordinator and let it handle that itself